### PR TITLE
Add Cmder, note that Babun is no longer maintained.

### PR DESCRIPTION
**Pull request by rwilliams**
Fixes https://github.com/sirredbeard/Awesome-WSL/issues/23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -293,7 +293,8 @@ An X server running on Windows is required for running Linux GUI apps on Windows
 
 - [Bash](https://www.gnu.org/software/bash/) - Bash is the GNU Project's shell. Bash is the Bourne Again SHell. Bash is an sh-compatible shell that incorporates useful features from the Korn shell (ksh) and C shell (csh).
 - [Cygwin](https://cygwin.com) - Cygwin is a Unix-like environment and command-line interface for Microsoft Windows.
-- [Babun](http://babun.github.io) - A Linux-like console on a Windows host. ![github project][githublogo]
+- [Cmder](https://cmder.net/) - A very nice console emulator built on ConEmu. ![github project][githublogo]
+- [Babun](http://babun.github.io) - [PROJECT DISCONTINUED] A Linux-like console on a Windows host. ![github project][githublogo]
```
Comment (rwilliams) [inline README.md]: Should Babun be removed all together?

Comment (sirredbeard) [reply README.md]: Yes, I think so. Awesome lists are supposed to be the best of the best.

**Comment by sirredbeard** [comment]
Remove Babun completely and I will merge. Thank you so much for this.

**Comment by rwilliams** [comment]
Updated

**Pull request merged**